app/assets/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,17 @@
+.sc-TakeoverHeader {
+
+  &-content {
+    @include display(flex);
```

Comment:
`.sc-TakeoverHeader-content` already has `display: flex;` I don't _think_ we need `justify-content: space-between;` here, either. The component should do this by default.